### ISSUE TITLE
Matomo real time map for download.kiwix.org is broken

### ISSUE THREAD

**Issue by kelson42** [kiwix/maintenance#66]
https://stats.kiwix.org/index.php?module=CoreHome&action=index&idSite=2&period=day&date=yesterday&updated=1#?idSite=2&period=day&date=yesterday&category=Dashboard_Dashboard&subcategory=1

You know why?

**Comment by florentk** [comment]
I have errors 500 on API call  with idSite=2 . I can't see the Matomo logs

**Comment by florentk** [comment]
@kelson42 It's a memory problem : 
```
Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes) in /var/www/html/libs/Zend/Db/Statement/Pdo.php on line 290
```

**Comment by kelson42** [comment]
@florentk https://matomo.org/docs/setup-auto-archiving/#increase-php-memory-limit

**Comment by florentk** [comment]
fix in b4e2982
